python/ray/tests/test_mldataset.py

```diff
@@ -121,6 +122,20 @@ def test_union(ray_start_regular_shared):
     assert ds.batch_size == 0
 
 
+def test_from_modin(ray_start_regular_shared):
```

Comment:
Is it ever tested in our CI?

Reply:
Maybe we should add modin to our test requirements.txt?

Reply:
(or we can just move a test to the modin CI pipeline?)

Reply:
It is not tested yet. I was not sure about adding Modin to the test requirements because I have seen some pushback on that in the past. I am ok to do whichever makes the most sense to the Ray team.

Reply:
I think it is ok to add Modin to the test requirements.